ingest/build.gradle

```diff
@@ -10,14 +10,13 @@ plugins {
 }
 
 dependencies {
-    implementation(Dependencies.json())
+    implementation(Dependencies.springBootStarterWeb())
     implementation(Dependencies.commonsIO())
     implementation(Dependencies.ingestAPI())
     implementation(Dependencies.transformAPI())
     implementation(Dependencies.awsS3())
-    implementation platform((Dependencies.awsBom()))
```

Comment:
❓ Why remove these dependencies?

Reply:
We don't need them, right?

Reply:
We only need it if we add more AWS sdk dependencies. It ensure they are all compatible version.

Reply:
Thanks for the explanation. It is not needed right now, so I think we're good to remove it to keep our dependencies light and maintainable.